README.md

```diff
@@ -127,6 +124,24 @@ If you already have an existing Angular project on GitHub, skip step 1 and 2.
 5. Your project should be available at `https://<username>.github.io/<repositoryname>`.  
    Learn more about GitHub pages on the [official website](https://pages.github.com/).
 
+## ⚙️ Installation <a name="installation"></a>
+
+`angular-cli-ghpages` can be installed via `ng add`.
+This will install the NPM package and add the necessary `deploy` configuration to your `angular.json` file.
+
+```sh
+ng add angular-cli-ghpages
+```
+
+The `deploy` config will be added for the specified `defaultProject`.
+If there is no `defaultProject` set and there is only one project in your workspace, this project will be used.
+
+If you have multiple projects in one workspace, you can manually define the project name:
+
+```sh
+ng add angular-cli-ghpages --project MYPROJECTNAME
```

Comment:
For later, it would be cool if we would have an interactive choice during installation.